gpt_engineer/chat_to_files.py

```diff
@@ -66,11 +67,11 @@ def to_files(chat: str, workspace: DB):
     workspace : DB
         The workspace to add the files to.
     """
-    workspace["all_output.txt"] = chat  # TODO store this in memory db instead
+    dbs.memory["all_output.txt"] = chat
```

Comment:
Up to you if we address this now, but:

I don't think we should do this in this function. But rather do it in the same place this function is called.

Then to_files becomes much simpler and easier to reason about!

(We could also use pycharm to refactor and let call it "to_files_and_memory", which in turns calls "to_files" that only takes the workspace)

Reply:
Agree & changed